templates/software-partnerships/index.html

```diff
@@ -0,0 +1,389 @@
+            <p>Join the embedding programme and start an embedding partnership with Canonical.</p>
+            <p>
+              <a href="/embedding">Learn more about the embedding programme&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-9 col-start-large-4 col-medium-3 col-start-medium-4">
+      <div class="p-block">
+        <hr class="p-rule">
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading-icon__title p-heading--5">Promote your software as "Validated with Canonical"</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>Validate against Canonical offerings (including Ubuntu, OpenStack and Kubernetes), with testing and tuning for optimal software performance.</p>
+            <p>Canonical also offers support assurance for the LTS releases of Ubuntu for partner applications.</p>
+            <p>
+              <a href="/hpc/contact-us?product=hpc-isv" class="p-button">Contact us for validation</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <hr class="p-rule">
+  </div>
+  <div class="row u-sv3">
+    <div class="col-3 col-medium-3 u-sv3">
+      <h2>Resources</h2>
+    </div>
+    <div class="col-9 col-medium-3">
+      <div class="row">
+        <div class="p-block">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading-icon__title p-heading--5">Product documentation</h3>
+            </div>
+            <div class="col-6">
+              <p>
+                <a href="https://juju.is/docs">Read the Juju documentation</a> for more details.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="p-block">
+          <hr class="p-rule">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading-icon__title p-heading--5">Community support</h3>
+            </div>
+            <div class="col-6">
+              <p>Join our community forums for <a href="https://discourse.ubuntu.com/">Ubuntu</a> or <a href="https://chat.charmhub.io/charmhub/channels/community-workshops">Juju</a>. Everyone is welcome!</p>
+            </div>
+          </div>
+        </div>
+        <div class="p-block">
+          <hr class="p-rule">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading-icon__title p-heading--5">Community support</h3>
+            </div>
+            <div class="col-6">
+              <ul class="p-list u-no-margin--bottom" id="software-partnerships-container">
+                <div><i class="p-icon--spinner u-animation--spin">Loading...</i></div>
```

Comment:
Nice to have, Element div is not allowed as a child of element ul in this context.